source/Ecs.CSharp.Benchmark/Program.cs

```diff
@@ -14,21 +17,34 @@
 
 BenchmarkSwitcher benchmark = BenchmarkSwitcher.FromTypes(new[]
 {
-    typeof(CreateEntityWithOneComponent),
-    typeof(CreateEntityWithTwoComponents),
-    typeof(CreateEntityWithThreeComponents),
-
     typeof(SystemWithOneComponent),
     typeof(SystemWithTwoComponents),
     typeof(SystemWithThreeComponents),
 
-    typeof(SystemWithTwoComponentsMultipleComposition)
+    typeof(SystemWithTwoComponentsMultipleComposition),
+
+    //Moving lighter tests to the back makes the estimated time display more reliable 
+    typeof(CreateEntityWithOneComponent),
+    typeof(CreateEntityWithTwoComponents),
+    typeof(CreateEntityWithThreeComponents),
 });
 
-IConfig configuration = DefaultConfig.Instance.WithOptions(ConfigOptions.DisableOptimizationsValidator);
+
+IConfig configuration = DefaultConfig.Instance
+    .WithOptions(ConfigOptions.DisableOptimizationsValidator)
+    .WithCapabilityExclusions();
 
 if (args.Length > 0)
 {
+    // There's no orderer commandline arg, so let's pretend there is one.
+    if (args.Contains("--ranking"))
+    {
+        List<string> argList = args.ToList();
+        argList.Remove("--ranking");
+        args = argList.ToArray();
+        configuration = configuration.WithOrderer(new DefaultOrderer(SummaryOrderPolicy.FastestToSlowest));
```

Comment:
I see this is an active or not settings in [benchmarkdotnet](https://mawosoft.github.io/BenchmarkDotNet/articles/configs/orderers.html). I'm thinking that maybe it would be better to stick to their philosophy (and what I'm doing for the [CheckCacheMisses](https://github.com/Doraku/Ecs.CSharp.Benchmark/blob/master/source/Ecs.CSharp.Benchmark/Ecs.CSharp.Benchmark.csproj#L16) [attribute](https://github.com/Doraku/Ecs.CSharp.Benchmark/blob/master/source/Ecs.CSharp.Benchmark/CreateEntityWithOneComponent/_CreateEntityWithOneComponent.cs#L7) for exemple 🤔). What do you think?

Reply:
Sure, just got home from vacation, will do it soon :)
Thank you for reviewing!

Reply:
I think we can go with the #define (to make starting conditions more uniform), but I'd still use it at the configuration level, rather than at the Attribute level. (BenchmarkDotNet encourages both).

Working on it right now..

Reply:
```cs
IConfig configuration = DefaultConfig.Instance
    .WithOptions(ConfigOptions.DisableOptimizationsValidator)
    .WithCapabilityExclusions();

#if RANK_RESULTS
    configuration = configuration.WithOrderer(new BenchmarkDotNet.Order.DefaultOrderer(BenchmarkDotNet.Order.SummaryOrderPolicy.FastestToSlowest));
#endif

if (args.Length > 0)
{
    benchmark.Run(args, configuration);
}
else
{
    benchmark.RunAll(configuration);
}
```

Reply:
Let me know what you think, otherwise I can just remove the change entirely. For my own workflow, the define is certainly less convenient than a command line parameter, but it's your repo and you also run the benchmark in its full scale, so this is what counts.